brave/brave-core
# Disable v2 ext on toggle turning off. Added the `Remove` button for disabled extensions.

Resolves https://github.com/brave/brave-browser/issues/42608

<details>
<summary> Video </summary>

https://github.com/user-attachments/assets/68a568a9-1de5-4918-8364-035edfbec7c1

</details>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open `brave://settings/extensions/v2`
2. Toggle on an extension (it will be installed)
3. Toggle off the extension, check `Remove` button appears and extension is disabled (on `brave://extension`)
4. Click `Remove`, check `Remove` button disappears and extension is uninstalled (on `brave://extension`)
